service/controller/resource/statuscondition/create_test.go

```diff
@@ -6,30 +6,19 @@ import (
 	"testing"
 
 	infrastructurev1alpha2 "github.com/giantswarm/apiextensions/v3/pkg/apis/infrastructure/v1alpha2"
-	"github.com/giantswarm/apiextensions/v3/pkg/apis/release/v1alpha1"
 	"github.com/giantswarm/k8sclient/v5/pkg/k8sclienttest"
 	"github.com/giantswarm/micrologger/microloggertest"
 	apiv1alpha2 "sigs.k8s.io/cluster-api/api/v1alpha2"
 
-	v1 "k8s.io/api/core/v1"
-
 	"github.com/giantswarm/cluster-operator/v3/service/internal/recorder"
-	"github.com/giantswarm/cluster-operator/v3/service/internal/releaseversion"
-	tcunittest "github.com/giantswarm/cluster-operator/v3/service/internal/tenantclient/unittest"
 	"github.com/giantswarm/cluster-operator/v3/service/internal/unittest"
 )
 
-type nodeConfig struct {
-	replicas      int
-	readyReplicas int
-}
-
 func TestComputeClusterStatusConditions(t *testing.T) {
```

Comment:
I did not want to remove this test as it's still useful for including all the different computation checks. However, I simplified and removed the parts that are now not needed anymore.